hoodie-client/src/main/java/com/uber/hoodie/index/bloom/HoodieBloomIndex.java

```diff
@@ -159,14 +161,11 @@ public HoodieBloomIndex(HoodieWriteConfig config) {
   }
 
   /**
-   * The index lookup can be skewed in three dimensions : #files, #partitions, #records
-   * <p>
-   * To be able to smoothly handle skews, we need to compute how to split each partitions into
-   * subpartitions. We do it here, in a way that keeps the amount of each Spark join partition to <
-   * 2GB.
-   * <p>
-   * If {@link com.uber.hoodie.config.HoodieIndexConfig#BLOOM_INDEX_PARALLELISM_PROP} is specified
-   * as a NON-zero number, then that is used explicitly.
+   * The index lookup can be skewed in three dimensions : #files, #partitions, #records <p> To be able to smoothly
```

Comment:
see if you can avoid these reformatting changes? set some auto formatting to "changed lines only" or somethjing?